themes/theme-default/src/client/components/global/CodeGroup.ts

```diff
@@ -127,35 +157,42 @@ export const CodeGroup = defineComponent({
         })
       }
 
-      return h('div', { class: 'code-group' }, [
-        h(
-          'div',
-          { class: 'code-group-nav', role: 'tablist' },
-          items.map((vnode, i) => {
-            const isActive = i === activeIndex.value
-            return h(
-              'button',
-              {
-                ref: (element) => {
-                  if (element) {
-                    tabRefs.value[i] = element as HTMLButtonElement
-                  }
-                },
-                class: {
-                  'code-group-nav-tab': true,
-                  'active': isActive,
+      return h(
+        'div',
+        {
+          class: 'code-group',
+          ref: (el) => (groupRef.value = el as HTMLDivElement),
```

Comment:
How does a direct assign `el: groupRef` not working?

Reply:
I forgot that I could write like this. 😂